app/src/main/java/com/dessalines/thumbkey/utils/Types.kt

```diff
@@ -94,54 +94,55 @@ enum class ThemeColor(private val stringId: Int) {
+    MessageEaseES("MessageEase español", 17),
     MessageEaseENSymbols("MessageEase english with symbols", 16),
-    MessageEaseHE("MessageEase עברית", 17),
-    MessageEaseDE("MessageEase deutsch", 23),
-    ThumbKeyNOv1("Thumb-Key norsk v1", 24),
-    ThumbKeyDEv2MultiLingual("Thumb-Key deutsch v2 multilingual", 25),
-    ThumbKeyKAv1("Thumb-Key kartuli ena v1", 26),
-    ThumbKeyIDv1("Thumb-Key bahasa indonesia with symbols v1", 27),
-    MessageEaseFR("MessageEase français", 28),
-    MessageEaseRUSymbols("MessageEase русский with Symbols", 29),
-    T9v1("T9 v1", 30),
-    ThumbKeyJAv1Hiragana("Thumb-Key japanese Hiragana v1", 31),
-    ThumbKeyJAv1Katakana("Thumb-Key japanese Katakana v1", 32),
-    ThumbKeyFRv2("Thumb-Key français v2", 33),
-    ThumbKeySVv1("Thumb-Key svenska v1", 34),
-    ThumbKeyTRv1("Thumb-Key Türkçe v1", 35),
-    TypeSplitENv2("Type-Split english v2", 36),
-    TypeSplitESv1("Type-Split español v1", 37),
-    TypeSplitDEv1("Type-Split deutsch v1", 38),
-    TypeSplitFRv1("Type-Split français v1", 39),
-    TypeSplitITv1("Type-Split italiano v1", 40),
-    TypeSplitPTv1("Type-Split português v1", 41),
-    TypeSplitPLv1("Type-Split polski v1", 42),
-    TwoHandsENv1("v. Two Hands english v1", 43),
-    WideLayoutENProgrammer("Wide Layout English Programmer", 44),
-    ThumbKeyHUv1("Thumb-Key magyar nyelv v1", 45),
-    ThumbKeyESEOv1("Thumb-Key español esperanto v1", 46),
-    MessageEaseIT("MessageEase italiano", 47),
-    ThumbKeyENv4Multi("Thumb-Key english v4 (EN+SK multi)", 48),
+    MessageEaseHE("MessageEase עברית", 18),
+    MessageEaseDE("MessageEase deutsch", 24),
+    ThumbKeyNOv1("Thumb-Key norsk v1", 25),
+    ThumbKeyDEv2MultiLingual("Thumb-Key deutsch v2 multilingual", 26),
+    ThumbKeyKAv1("Thumb-Key kartuli ena v1", 27),
+    ThumbKeyIDv1("Thumb-Key bahasa indonesia with symbols v1", 28),
+    MessageEaseFR("MessageEase français", 29),
+    MessageEaseRUSymbols("MessageEase русский with Symbols", 30),
+    T9v1("T9 v1", 31),
+    ThumbKeyJAv1Hiragana("Thumb-Key japanese Hiragana v1", 32),
+    ThumbKeyJAv1Katakana("Thumb-Key japanese Katakana v1", 33),
+    ThumbKeyFRv2("Thumb-Key français v2", 34),
+    ThumbKeySVv1("Thumb-Key svenska v1", 35),
+    ThumbKeyTRv1("Thumb-Key Türkçe v1", 36),
+    TypeSplitENv2("Type-Split english v2", 37),
+    TypeSplitESv1("Type-Split español v1", 38),
+    TypeSplitDEv1("Type-Split deutsch v1", 39),
+    TypeSplitFRv1("Type-Split français v1", 40),
+    TypeSplitITv1("Type-Split italiano v1", 41),
+    TypeSplitPTv1("Type-Split português v1", 42),
+    TypeSplitPLv1("Type-Split polski v1", 43),
+    TwoHandsENv1("v. Two Hands english v1", 44),
+    WideLayoutENProgrammer("Wide Layout English Programmer", 45),
+    ThumbKeyHUv1("Thumb-Key magyar nyelv v1", 46),
+    ThumbKeyESEOv1("Thumb-Key español esperanto v1", 47),
+    MessageEaseIT("MessageEase italiano", 48),
+    ThumbKeyENv4Multi("Thumb-Key english v4 (EN+SK multi)", 49),
```

Comment:
Change back all these to what they were, and add yours at the end, with a higher number. The app sorts them by name anyway.